### PR TITLE
[ETL-425] Bootstrap trigger diffs upon union of sharedschema and archivemetadata

### PR DESCRIPTION
**Pull request by philerooski**
* Bootstrap trigger script now accepts a second parquet dataset and performs a union of the first and second dataset's "diff field" before taking a diff with the records on Synapse.
* Crontab updated to use both sharedschema and archivemetadata datasets.

### CODE REVIEW

File: src/scripts/bootstrap_trigger/bootstrap_trigger.py [hunk 1]
```diff
@@ -226,8 +234,16 @@ def main():
                 dataset_uri=args.diff_s3_uri,
                 aws_session=aws_session,
                 columns=[args.diff_parquet_field])
+        already_processed_records = parquet_dataset[args.diff_parquet_field]
+        if args.diff_s3_uri_2 is not None and args.diff_parquet_field_2 is not None:
+            parquet_dataset_2 = get_parquet_dataset(
+                    dataset_uri=args.diff_s3_uri_2,
+                    aws_session=aws_session,
+                    columns=[args.diff_parquet_field_2])
+            already_processed_records = already_processed_records.append(
```
Comment (rxu17) [inline src/scripts/bootstrap_trigger/bootstrap_trigger.py:243]: Nit: Just realized this but there is a `FutureWarning: The series.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.` for the `append` method for pandas

```suggestion
            already_processed_records = pd.concat([already_processed_records, parquet_dataset_2[args.diff_parquet_field]])
```

**Review by rxu17 — APPROVED** [review]
LGTM, one minor comment!

File: src/scripts/bootstrap_trigger/bootstrap_trigger.py (hunk 1 above)
Comment (philerooski) [reply src/scripts/bootstrap_trigger/bootstrap_trigger.py]: There was also a bug on that line so I fixed it.

Comment (rxu17) [reply src/scripts/bootstrap_trigger/bootstrap_trigger.py]: That extra `]` :')

Comment (philerooski) [reply src/scripts/bootstrap_trigger/bootstrap_trigger.py]: Oh, I meant in my code. I used the wrong command line argument.

Comment (rxu17) [reply src/scripts/bootstrap_trigger/bootstrap_trigger.py]: Wow, missed that as well. Nice catch :')

**Pull request merged**